docs/Style-Guide.md

```diff
@@ -76,29 +114,33 @@ $message-radius: 1px;
 
 If there isn't a SCSS variable for that, the solution is to override directly the classes. Do that **after** the module import. When overriding a class, ask yourself if that should be modified everywhere by default or if it should be a new modifier.
 
-If it's a new modifier, add `.gi-` prefix. That brings at least 2 main benetifs:
+If it's a new modifier, add `.gi-` prefix. That brings at least 2 main benefits:
 - Easily understand if that code is Bulma's or ours.
 - Prevent Bulma's naming conflicts with our code on future updates.
 
 Let's see an example:
 
-Bulma's `.message` default `border-width` value is `0 0 0 3px`. We want it to be just `1px` on a particular view, so let's add a new modifier.
-
 ```scss
 $message-radius: 1px;
 
 @import "../../node_modules/bulma/sass/components/message";
 
 .message-body {
-  &.gi-is-box {
-    border-width: 1px;
+  // The default `border-width` value is `0 0 0 3px`.
+  // We want it to be `1px` everywhere, so we add a new modifier:
+  border-width: 1px;
+
+  // The default border-style is solid,
+  // We want to dashed it just on a particular view, so we add a new modifier:
+  &.gi-is-unfilled {
+    border-style: dashed
   }
 }
 ```
 
-When's used it's painless to differentiate our code from Bulma's!
-
 ```html
+```html
```

Comment:
Line 142 is a duplicate and should be deleted

Reply:
removed